my_game.py

```diff
@@ -183,6 +183,12 @@ def on_draw(self):
             SCREEN_HEIGHT - 20,  # Y positon
             arcade.color.WHITE   # Color of text
         )
+        arcade.draw_text(
+            "LIVES: {}".format(PLAYER_LIVES),  # Text to show
```

Comment:
That's a constant configuration! Where are the playe's lifes? In the MyGame or the Player?